Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.5.1'
+ruby '2.5.3'
```

Comment:
Please also update [CircleCI config](https://github.com/paritytech/clachair/blob/ab4ea9c43793a20c9e05d2c949929121bac0deec/.circleci/config.yml#L10) to use this exact version of Ruby.